osu.Game/OsuGame.cs

```diff
@@ -290,6 +297,8 @@ public void CloseAllOverlays(bool hideToolbar = true)
             if (hideToolbar) Toolbar.Hide();
         }
 
+        public void ChangeLogoDepth(bool inFrontOfFooter) => ScreenContainer.ChangeChildDepth(logoContainer, inFrontOfFooter ? float.MinValue : 0);
```

Comment:
I dunno about this. I'd rather just see it using `Parent` access similar to other usages in `LogoTrackingContainer`...

Reply:
I don't follow what you mean by "`Parent` access".

Reply:
`Parent.ChangeChildDepth` rather than `OsuGame` method.

Reply:
Well...the drawable which its depth is being changed here is the parent of `OsuLogo`, meaning that this method is the equivalent of `logo.Parent.Parent.ChangeChildDepth(logo.Parent, float.MinValue)`, and not to mention that I can't access `ChangeChildDepth` directly without direct-casting to `Container` first. That's why I settled for an `OsuGame` method to ease my head.

Reply:
Right. It's still not great. See if you can find a nicer way of doing this, else I'll take a look during next review pass.

Reply:
I have already spent hours before making this PR trying to figure out how to fix this issue of the logo depth being a nuisance in the user's experience, but I'll try once more and see if I have a clever idea this time.